public/locales/es/common.json

```diff
@@ -124,6 +124,11 @@
     "dark": "Mono Dark",
     "explorer": "Explorer",
     "light": "Mono Light",
-    "system-prefs": "Sistema"
+    "system-prefs": "Sistema",
+    "custom": "Personalizado",
```

Comment:
I will live themes: {} just for the themes names, and move `custom` and `gallery` keys to `modals.json`.
Something like:
```
"community-modal": {
    "title: ".....",
    "community: "....",
    "custom": "Personalizado"
}
```

Reply:
ok! any action needed on my end?

Reply:
@maraoz Can you pull develop and make the change I mentioned, didn't have any time to do it!

Reply:
@turtlemoji ready for review, I did as you requested!

same as before, I tested this manually locally by switching between English and Spanish. Seems to work